sphinx/source/docs/quickstart.rst

```diff
@@ -6,374 +6,585 @@ Quickstart
 Introduction
 ============
 
-Bokeh is a Python interactive visualization library that targets modern web browsers
-for presentation providing elegant, concise construction of novel graphics with
-high-performance interactivity over very large or streaming datasets in quick and
-easy way.
+Bokeh is a Python interactive visualization library that targets modern web
+browsers for presentation providing elegant, concise construction of novel
+graphics with high-performance interactivity over very large or streaming
+datasets in quick and easy way.
 
-Offering both powerful and flexible features to enable very advanced customizations
-in one hand and simplicity on the other Bokeh exposes different interface levels
-to the users:
+Offering both powerful and flexible features to enable very advanced
+customizations in one hand and simplicity on the other Bokeh exposes different
+interface levels to the users:
 
-* a low Level (and more flexible) glyph interface
-* an intermediate level interface called plotting
-* a high level interface that can be used to build complexs plot in a simple way.
+* a *low-level* |bokeh.models| interface that provides the most flexibility
+  to application developers.
+* an *intermediate-level* |bokeh.plotting| interface that is centered
+  around composing visual glyphs.
```

Comment:
What about being more specific about turning this into the `glyphs` interface? If we start calling it `glyphs` when answering questions we need to be more specific otherwise users would not find a clear reference in the docs..